src/mocks/index.js

```diff
@@ -147,17 +150,21 @@ mock.get(
 );
 
 mock.put('/veilarbaktivitet/api/aktivitet/:aktivitetId', failOrGetResponse(aktivitetFeilet, oppdaterAktivitet));
+mock.post('/veilarbaktivitet/api/aktivitet/:aktivitetId', failOrGetResponse(aktivitetFeilet, oppdaterAktivitet));
```

Comment:
Skal den være post og ikke put?

Reply:
Vel, jeg registrerte i hvert fall at post ble kalt når jeg brukte demoen, så jeg måtte legge til denne for å få kalt mock-koden. Det kan hende put brukes andre steder jeg ikke oppdaget.